mscore/scoreview.h

```diff
@@ -174,6 +175,7 @@ class ScoreView : public QWidget, public MuseScoreView {
       virtual void mouseDoubleClickEvent(QMouseEvent*);
 
       virtual void keyPressEvent(QKeyEvent*) override;
+      void _keyPressEvent(QKeyEvent* ev);
```

Comment:
I was trying things out in the beginning. I forgot to delete this, I'll delete it now.